### PR TITLE
update: add org permissions and roles

### PR DESCRIPTION
**Pull request by staceysalamon-aiven**
## Checklist

- [ ] The first paragraph of the page is on one line.
- [ ] The other lines have a line break at 90 characters.
- [ ] I checked the output.
- [ ] I applied the [style guide](styleguide.md).
- [ ] My links start with `/docs/`.

### CODE REVIEW

**Comment by staceysalamon-aiven** [comment]
Preview: https://staceys-doc-1200.aiven-docs.pages.dev/docs/platform/concepts/permissions

**Review by rsalevsky — APPROVED** [review]

File: docs/platform/howto/manage-permissions.md [hunk 1]
```diff
@@ -5,9 +5,64 @@ title: Manage permissions
 import ConsoleLabel from "@site/src/components/ConsoleIcons"
 import {ConsoleIcon} from "@site/src/components/ConsoleIcons"
 
-You can give users and groups access to a project and the services in it by granting them roles and permissions for that project.
+You can grant [organzation users](/docs/platform/howto/manage-org-users), [application users](/docs/platform/concepts/application-users), and [groups](/docs/platform/howto/manage-groups) access at the organization and project level through [roles and permissions](/docs/platform/concepts/permissions).
```
Comment (harshini-rangaswamy) [inline docs/platform/howto/manage-permissions.md:8]:
```suggestion
You can grant [organization users](/docs/platform/howto/manage-org-users), [application users](/docs/platform/concepts/application-users), and [groups](/docs/platform/howto/manage-groups) access at the organization and project level through [roles and permissions](/docs/platform/concepts/permissions).
```

File: docs/platform/howto/make-super-admin.md [hunk 2]
```diff
@@ -4,20 +4,13 @@ title: Super admin
 
 import ConsoleLabel from "@site/src/components/ConsoleIcons"
 
-The super admin role has full access to an organization, its billing and settings, and all its organizational units, projects, and services.
+The super admin role is a special role that has unrestricted access to an organization and all of is resources. This role should be limited to as few users as possible for organization setup and emergency use. For daily administrative tasks, assign users the [organization admin role](/docs/platform/concepts/permissions) instead. Aiven also highly recommends enabling [two-factor authentication](/docs/platform/howto/user-2fa) for super admin.
```
Comment (harshini-rangaswamy) [inline docs/platform/howto/make-super-admin.md:7]:
```suggestion
The super admin role is a special role that has unrestricted access to an organization, and all of its resources. This role should be limited to as few users as possible for organization setup and emergency use. For daily administrative tasks, assign users the [organization admin role](/docs/platform/concepts/permissions) instead. Aiven also highly recommends enabling [two-factor authentication](/docs/platform/howto/user-2fa) for super admin.
```

File: docs/platform/concepts/permissions.md [hunk 3]
```diff
@@ -8,58 +8,69 @@ To give users access to projects and services in your organizations, you grant t
...
+Principals are
+[organization users](/docs/platform/howto/manage-org-users),
+[application users](/docs/platform/concepts/application-users),
+and [groups](/docs/platform/howto/manage-groups).
 
+You can grant access to principals at the organization and project level.
 You can
-[grant access to principals at the project level](/docs/platform/howto/manage-permissions).
-You can also [add users to services](/docs/platform/howto/create_new_service_user).
+[add users to services](/docs/platform/howto/create_new_service_user).
```
Comment (harshini-rangaswamy) [inline docs/platform/concepts/permissions.md:18]: Could you clarify the purpose of including 'add users to services' here? It feels slightly disconnected from the previous sentence. Is it meant to complement granting access or highlight a separate action?

**Review by harshini-rangaswamy — APPROVED** [review]
Looks good to me! Noticed some minor typos.

**Pull request merged**